Packs/GenericMCP/pack_metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "Generic MCP",
+    "description": "Generic Module Content Pack for managing and extending integration capabilities",
```

Comment:
this description needs a refinement

Reply:
@dantavori  What do you think about this:
`"This pack contains Universal integration for Model Context Protocol (MCP), allowing seamless connectivity to an MCP servers"`

Reply:
"This pack contains a generic integration with Model Context Protocol (MCP) servers, allowing seamless connectivity to an MCP servers"